google/cloud/aiplatform/initializer.py

```diff
@@ -44,7 +45,6 @@ class _Config:
 
     def __init__(self):
```

Comment:
This class most likely needs a helper method `_reset_metadata_tracking` that sets `experiment` and `run` to `None`. This should be called if `project` or `location` is changed.

Reply:
Synced offline with @jialuzh, he is planning on taking care of these things in follow up PRs. Would that be ok?

Reply:
SGTM